requirements/py310.yml

```diff
@@ -11,7 +11,8 @@ dependencies:
 
 # Core dependencies.
   - iris>=3.0.2
-  - python-eccodes
+  - python-eccodes>=1.6.1
```

Comment:
? not needed ?
as above

Reply:
Yup, it is, see [comment](https://github.com/SciTools/iris-grib/pull/361#discussion_r1394813547) above.